debian/cephadm.postinst

```diff
@@ -25,7 +25,12 @@ case "$1" in
        # 1. create user if not existing
        if ! getent passwd | grep -q "^cephadm:"; then
          echo -n "Adding system user cephadm.."
-         adduser --quiet --system --disabled-password --gecos 'cephadm user for mgr/cephadm' --shell /bin/bash cephadm 2>/dev/null || true
+         adduser --quiet \
+                 --system \
+                 --disabled-password \
+                 --home /home/cephadm \
+                 --gecos 'cephadm user for mgr/cephadm' \
```

Comment:
Hey, I am new to ceph, I was trying to install cephamd on Debian bookworm and ran into the problem this PR solves.

I was going through the Debian docs and found that they are [going to remove `--gecos` option in the next release](https://manpages.debian.org/bookworm/adduser/adduser.8.en.html#comment). Instead, `--comment` should be used. I see that for the other postinst script `--comment` is already used.

Does it make sense to use the new option for this script too as a part of this PR or should it be solved in a different ticket?

Reply:
@Sitiritis hi Tymur, thank you for the reminder. since this change is being tested in lab, and i hope that this fix can be backported sooner. and and `--gecos` has not been deprecated, so guess we can live with it a little bit longer. but i will create another PR to s/`--gecos`/`--comment`/ to be more future-proof. so, "in a different ticket" or pull request. in general, we file a ticket so that it can be tracked and the corresponding fixes can be backported. if the `--gecos` does not cause any real-world issues yet, we could dispense a ticket on tracker.

Reply:
@Sitiritis since our CI is still using ubuntu:jammy, and the `adduser` shipped along with it does not support `--comment` yet, i am not using `--comment` and since `--gecos` is going to be removed after bookworm, as you point out. i am adding a commit to use `usermod` directly. once we have the luxury of dropping the support of release before debian/trixie or ubuntu/mantic. we can start using `adduser --comment`. to be more future-proof, i will add a commit in this PR to switch to `usermod`.

Reply:
Thank you :)
I just got access to ceph's redmine to create a ticket for this change, but it seems I don't need to do this anymore.